jerry-core/jerry-port.h

```diff
@@ -19,29 +20,26 @@
 #include <stdio.h>
 
 #ifdef __cplusplus
-# define EXTERN_C "C"
-#else /* !__cplusplus */
-# define EXTERN_C
+extern "C"
+{
 #endif /* !__cplusplus */
 
 /** \addtogroup jerry_port Jerry engine port
  * @{
  */
 
-/**
- * Target port functions for console output
- */
-extern EXTERN_C
-int jerry_port_logmsg (FILE *stream, const char *format, ...);
-
-extern EXTERN_C
-int jerry_port_errormsg (const char *format, ...);
-
-extern EXTERN_C
-int jerry_port_putchar (int c);
+  /**
+   * Target port functions for console output
+   */
+  int jerry_port_logmsg (FILE *stream, const char *format, ...);
+  int jerry_port_errormsg (const char *format, ...);
+  int jerry_port_putchar (int c);
 
 /**
  * @}
  */
 
 #endif /* !JERRY_API_H */
+#ifdef __cplusplus
+} /*  */
+#endif /* !__cplusplus */
```

Comment:
The order seems to be mixed up here. JERRY_PORT_H starts, extern C starts, JERRY_PORT_H ends, extern C ends.
And, lo, wrong closing comment! this is not JERRY_API_H...